sources/packages/web/src/components/common/InstitutionUserSummary.vue

```diff
@@ -72,137 +56,116 @@
       <Column :field="UserFields.Email" header="Email" sortable="true"></Column>
       <Column :field="UserFields.UserType" header="User Type">
         <template #body="slotProps">
-          <ul
-            class="no-bullets"
-            v-for="userType in slotProps.data.userType"
-            :key="userType"
-          >
-            <li>{{ userType }}</li>
-          </ul></template
-        ></Column
+          <!-- Get the first index of the array since the user will have admin or user type only. -->
+          {{ slotProps.data.userType[0] }}
+        </template></Column
```

Comment:
if we are not using the `no-bullets` custom class anywhere else, may be we can delete the style from the scss

Reply:
True, I will remove it.

Reply:
`no-bullets` style removed.